example/mqtt/src/main/java/org/apache/iotdb/mqtt/MQTTClient.java

```diff
@@ -35,7 +35,7 @@ public static void main(String[] args) throws Exception {
     BlockingConnection connection = mqtt.blockingConnection();
     connection.connect();
 
-    Random random = new Random();
+    ThreadLocalRandom random = ThreadLocalRandom.current();
```

Comment:
why?   Will performance be affected?

Reply:
https://github.com/apache/iotdb/issues/5880 
Threadlocalrandom's method nextseed() that generates random seeds in a multithreaded environment has higher performance, while random is generated through CAS' method